kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/internal/KotestEngineSystemProperties.kt

```diff
@@ -54,4 +54,9 @@ object KotestEngineSystemProperties {
     * Disable scanning the classpath for listeners with @AutoScan by setting this property to true
     */
    const val disableAutoScanClassPathScanning = "kotest.framework.classpath.scanning.autoscan.disable"
+
+   /**
+   *  If set -> filter testCases by this severity level and higher, else running all
+   */
+   const val severityPrefix = "kotest.severity"
```

Comment:
kotest.framework.test.severity is a more precise namespace

Reply:
done